src/components/reusable-ui/Card.jsx

```diff
@@ -0,0 +1,77 @@
+import styled from "styled-components"
+import { theme } from "../../theme/index.js"
+import PrimaryButton from "./PrimaryButton"
+
+export default function CardItem({ imageSource, title, price }) {
+  return (
+    <CarteItemStyled>
```

Comment:
```suggestion
    <CardStyled>
```

en anglais et pas besoin de mettre le "item"